src/Symfony/Component/PropertyInfo/PropertyInfoExtractor.php

```diff
@@ -18,33 +18,18 @@
  */
 class PropertyInfoExtractor implements PropertyInfoExtractorInterface
 {
-    /**
-     * @var PropertyListExtractorInterface[]
-     */
     private $listExtractors;
-
-    /**
-     * @var PropertyTypeExtractorInterface[]
-     */
     private $typeExtractors;
-
-    /**
-     * @var PropertyDescriptionExtractorInterface[]
-     */
     private $descriptionExtractors;
-
-    /**
-     * @var PropertyAccessExtractorInterface[]
-     */
     private $accessExtractors;
 
     /**
-     * @param PropertyListExtractorInterface[]        $listExtractors
-     * @param PropertyTypeExtractorInterface[]        $typeExtractors
-     * @param PropertyDescriptionExtractorInterface[] $descriptionExtractors
-     * @param PropertyAccessExtractorInterface[]      $accessExtractors
+     * @param iterable|PropertyListExtractorInterface[]        $listExtractors
```

Comment:
`iterable` is not a valid keyword, even according to the (still) [draft PSR-5](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#abnf). See https://github.com/phpDocumentor/fig-standards/issues/141

But anyway I think it's redundant to write it like this, it should be simply `iterable<PropertyListExtractorInterface>`